### ISSUE TITLE
Trying to remove the integration card

### ISSUE THREAD

**Issue by guni12** [AlexxIT/SonoffLAN#1149]
I would like to remove the sonoff integration card. There doesn't seem to be a removal link/button for it or have I missed it somehow? 
If the integration is removed or not in HACS does not seem to affect the card.
![sonoff-red](https://user-images.githubusercontent.com/3921230/228004561-753ccbd1-bec5-4403-8091-8cd8ed964335.png)
Pressing the three dots only leads to documentation.
It would be nice to get rid of this red card.

**Comment by guni12** [comment]
Thank you for answering. Tried different browsers but with the same result unfortunately. 
![sonoff](https://user-images.githubusercontent.com/3921230/228202827-9c2c9f3b-f8ac-45b0-ba34-a3fab31eeb61.jpg)
Strange.

**Comment by AlexxIT** [comment]
Maybe you have something in the configuration.yaml file

**Comment by guni12** [comment]
No, nothing related to sonoff. A friend of ours have a similar issue with:
![skogsbo-red](https://user-images.githubusercontent.com/3921230/228236055-f83c36a5-ef4b-44d7-9a9b-ba5958b77d66.png)

**Comment by AlexxIT** [comment]
I think it's Hass problem and not related to this integration.
